README.md

```diff
@@ -464,12 +464,18 @@ With this you can customize the classes applied to various tree elements
 (`treeClass`, `emptyTreeClass`, `hiddenClass`, `nodesClass`, `handleClass`,
 `placeholderClass`, `dragClass`).
 
-In addition, you can modify whether nodes are collapsed by default
-(`defaultCollapsed`: default false). For example:
+In addition, you can modify whether or not nodes are collapsed by default
+(`defaultCollapsed`: default false).
```

Comment:
I'm comfortable with it being true by default. 

Yes, breaking other people's apps in an update is frowned upon, but I feel like in most implementations on ui-tree this provides a better experience, and it's extremely easy to override the new default.